codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyCodecUtil.java

```diff
@@ -305,6 +305,9 @@ static void validateHeaderName(CharSequence name) {
                 throw new IllegalArgumentException(
                         "name contains null character: " + name);
             }
+            if (c >= 'A' && c <= 'Z') {
+                throw new IllegalArgumentException("name must be all lower case.");
```

Comment:
If this is true we may need to back port this as a fix to other branches as well ?